dali/operators/reader/numpy_reader_op.cc

```diff
@@ -22,7 +22,30 @@
 
 namespace dali {
 
-void NumpyReaderCPU::TransposeHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &input) {
+static void CopyHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &input,
+                       ThreadPool &thread_pool, int min_blk_sz, int req_nblocks) {
+  auto out_ptr = static_cast<uint8_t*>(output.raw_mutable_data());
+  auto in_ptr = static_cast<const uint8_t*>(input.raw_data());
+  auto nelements = volume(input.shape());
+  auto nbytes = input.nbytes();
+  if (nelements <= min_blk_sz) {
+    thread_pool.AddWork([=](int tid) {
+      std::memcpy(out_ptr, in_ptr, nbytes);
+    }, nelements);
+  } else {
+    int64_t prev_b_start = 0;
+    auto task_sz = nelements / req_nblocks;
+    for (int b = 0; b < req_nblocks; b++) {
+      int64_t b_start = prev_b_start;
+      int64_t b_end = prev_b_start = nbytes * (b + 1) / req_nblocks;
+      thread_pool.AddWork([=](int tid) {
+        std::memcpy(out_ptr + b_start, in_ptr + b_start, (b_end - b_start));
+      }, task_sz);
```

Comment:
Use the actual size:
```suggestion
      }, (b_end - b_start) * kCopyCost);
```